### ISSUE TITLE
How to use?

### ISSUE THREAD

**Issue by NDevDiscord** [microsoft/fluentui-emoji#139]
Im on windows 10, im just want to use this, but how?

**Comment by thanone2** [comment]
I also want to know how to use this, but on android.

**Comment by nonetrix** [comment]
I assume there is some build script to generate a .tff file?

**Comment by kissu** [comment]
Import the SVG on your website/android or else and use it. 🤗

**Comment by thanone2** [comment]
> Import the SVG on your website/android or else and use it. 🤗 

Bruh... We want to replace the default android emojis not use these as stickers...

**Comment by kissu** [comment]
10s search: https://zfont-3.en.uptodown.com/android

**Comment by thanone2** [comment]
> 10s search: https://zfont-3.en.uptodown.com/android
> 
> This is not exactly the place to ask this anyway, they are providing the icons. It's other people's job to integrate them wherever they want them. Biggest part is all done.

Then why didn't you say that at first? +I don't want to root my phone

**Comment by kissu** [comment]
You [don't need to root](https://www.reddit.com/r/Windows11/comments/ykcl6x/comment/iuv5ccm/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button) for it.

Otherwise, I said what I thought was relevant in this case.
Makes sense to have such an app on Android, but could also be doable to just ship something like that on a custom keyboard like SwiftKey or like. Or any kind of app to wrap it around.

**Comment by NDevDiscord** [comment]
> Import the SVG on your website/android or else and use it. 🤗

im just want to have it on windows 10 pro 22h2

**Comment by kissu** [comment]
@NDevDiscord not sure about [how 22h2 for Win10](https://en.wikipedia.org/wiki/Windows_10_version_history#Version_22H2_(2022_Update)) and how it impacts this case but that kind of solution looks quite universal in its approach. On top of being released before the update, so I guess fully compatible too: https://youtu.be/I4a7h40ZrzQ

Again, 10s search on Google guys.

**Comment by kenmcd** [comment]
These emojis are used in the Segoe UI Emoji font.
The Flat color versions (SVGs) are used up to v1.33.
Those look different than v1.29, but do not have gradients.
Those are both still COLRv0 fonts - so it should work on Win10.
With v1.45 the font changed to COLRv1 - so it can have gradients.
So it does look different than v1.29 and v1.33.
Those are the SVGs in the Color folder.
I do not know if Win10 can handle the COLRv1 format of v1.45.
May be interesting to see if it does, but I have not seen anything suggesting it does.

Android can now handle COLRv1 emoji fonts.
Noto Emoji (color) is now COLRv1 format.
So my guess is that it could work.

No easy way to build a font from these SVGs (that would work the same way).

**Comment by nonetrix** [comment]
Yes build script to ttf is what we need ideally and maybe releases but that's just my opinion, you can download it from unofficial sources however which I suspect is just ripped from Windows 11. They probably have a internal build process they use but haven't released for whatever reason, hopefully not intentionally

**Comment by tetunori** [comment]
Hi, all.

This is Just FYI but I have just released (UNOFFICIALLY) TrueType ver and Webfont ver of "Fluent Emoji" based on the materials here.

Now that you can get them from the repository [fluent-emoji-webfont](https://github.com/tetunori/fluent-emoji-webfont).  
Especially, you can access to webfont(`woff2` files) and/or download `ttf` files from below:
- Webfont
  - [Fluent Emoji Color](https://tetunori.github.io/fluent-emoji-webfont/dist/FluentEmojiColor.css)
  - [Fluent Emoji Flat](https://tetunori.github.io/fluent-emoji-webfont/dist/FluentEmojiFlat.css)
  - [Fluent Emoji High Contrast](https://tetunori.github.io/fluent-emoji-webfont/dist/FluentEmojiHighContrast.css)

- TrueType font
  - [FluentEmojiColor.ttf](https://tetunori.github.io/fluent-emoji-webfont/dist/FluentEmojiColor.ttf)
  - [FluentEmojiFlat.ttf](https://tetunori.github.io/fluent-emoji-webfont/dist/FluentEmojiFlat.ttf)
  - [FluentEmojiHighContrast.ttf](https://tetunori.github.io/fluent-emoji-webfont/dist/FluentEmojiHighContrast.ttf)

For samples and the usage of these fonts, please refer to [the README file](https://github.com/tetunori/fluent-emoji-webfont) and [the release X post](https://x.com/tetunori_lego/status/1874403641169453287)

I hope my repository's result will be helpful for the future development of original fluent emoji scripts to build Webfonts and ttfs.
